### PR TITLE
Add endpoint documentation for recoverCumulusMessages

### PR DESCRIPTION
**Pull request by Jkovarik**
Adds documentation for DLQ recovery endpoint (CUMULUS-2869)

### CODE REVIEW

File: content/dead-letter-archive.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+The query uses the following optional parameters:
+
+| parameter | description |
+| --- | --- |
+| bucket | The bucket to read records from. Default the Core system bucket|
+| path | The S3 prefix (path) to read DLQ records from.  Defaults to `<stackName>/dead-letter-archive/sqs/`|
+
+```endpoint
+POST /deadLetterARchive/recoverCumulusMessages
```
Comment (kkelly51) [inline content/dead-letter-archive.md:15]: Can you make the R in archive lowercase.

File: content/dead-letter-archive.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## Recover cumulus messages
+
+Endpoint provides a mechanism for recovery of S3 sfEventSqsToDbRecords dead letter objects (created as described in the [Core Documentation](https://nasa.github.io/cumulus/docs/features/dead_letter_archive)).   The endpoint will invoke an async operation that will attempt to process all of the records in the specified location.
+
+The endpoint by default will process all records from the default storage location on the S3 system bucket `<stackName>/dead-letter-archive/sqs/`, however it is likely useful to process a subset of those granules by moving the records to process to another prefix on S3
```
Comment (markdboyd) [inline content/dead-letter-archive.md]:
```suggestion
The endpoint by default will process all objects from the default storage location on the S3 system bucket under the prefix of `<stackName>/dead-letter-archive/sqs/`. However, it is likely useful to process a subset of those objects, which you can do by moving the desired subset of objects to a new path on S3 and then specifying that new path using the `path` parameter to the endpoint request.
```

🐜

File: content/dead-letter-archive.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+## Recover cumulus messages
+
+Endpoint provides a mechanism for recovery of S3 sfEventSqsToDbRecords dead letter objects (created as described in the [Core Documentation](https://nasa.github.io/cumulus/docs/features/dead_letter_archive)).   The endpoint will invoke an async operation that will attempt to process all of the records in the specified location.
```
Comment (markdboyd) [inline content/dead-letter-archive.md]:
```suggestion
Endpoint provides a mechanism for recovery of S3 sfEventSqsToDbRecords dead letter objects (created as described in the [Core Documentation](https://nasa.github.io/cumulus/docs/features/dead_letter_archive)).   The endpoint will invoke an async operation that will attempt to process all of the objects in the specified location.
```

🐜

File: content/dead-letter-archive.md (hunk 2 above)
Comment (Jkovarik) [reply content/dead-letter-archive.md]: Disagree on s/objects/records.       While they're S3 *objects*, the thing that's being processed is a set of *records*, and from a user context the distinction matters.    We could say something like 'all records contained in the S3 objects from the default storage location', for example. 

The rest looks good.   Thoughts?

File: content/dead-letter-archive.md (hunk 1 above)
Comment (Jkovarik) [reply content/dead-letter-archive.md]: 👍🏻  thanks

File: content/dead-letter-archive.md (hunk 2 above)
Comment (Jkovarik) [reply content/dead-letter-archive.md]: try 51947ce  on for size, totally open to something else.

File: content/dead-letter-archive.md (hunk 3 above)
Comment (Jkovarik) [reply content/dead-letter-archive.md]: 12cd2ee -- git UI isn't committing this (conflicts?)

File: content/dead-letter-archive.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+## Recover cumulus messages
+
+Endpoint provides a mechanism for recovery of S3 sfEventSqsToDbRecords dead letter objects (created as described in the [Core Documentation](https://nasa.github.io/cumulus/docs/features/dead_letter_archive)). The endpoint will invoke an async operation that will attempt to process all of the objects in the specified location.
+
+The endpoint by default will process all all records contained in the S3 objects from the default storage location on the S3 system bucket under the prefix of `<stackName>/dead-letter-archive/sqs/`. However, it is likely useful to process a subset of those objects, which you can do by moving the desired subset of objects to a new path on S3 and then specifying that new path using the `path` parameter to the endpoint request.
```
Comment (markdboyd) [inline content/dead-letter-archive.md:5]:
```suggestion
The endpoint by default will process all records contained in the S3 objects from the default storage location on the S3 system bucket under the prefix of `<stackName>/dead-letter-archive/sqs/`. However, it is likely useful to process a subset of those objects, which you can do by moving the desired subset of objects to a new path on S3 and then specifying that new path using the `path` parameter to the endpoint request.
```

**Pull request merged**